advisor-go/src/quixo/simulation/simulation.go

```diff
@@ -16,35 +16,54 @@ func GetBestMoveForPlayer(board game.Board) Move {
 	return getBestMove(board, false)
 }
 
-// GetWorstMoveForPlayer return the move that will benefit the most to the opponent player
+// GetWorstMoveForPlayer return the move that will benefit the most for the opponent player
 func GetWorstMoveForPlayer(board game.Board) Move {
 	return getBestMove(board, true)
 }
 
+// GetMoveThatMakeMyOpponentLose return the move that will be the worst for the opponent
+func GetMoveThatMakeMyOpponentLose(board game.Board) Move {
+	isSmallerThan := func(int1 int, int2 int) bool { return int1 < int2 }
```

Comment:
I don't understand that ...?

Reply:
I change the getMove function to take a comparator. So I can get the board with the max score or the board with the min score.

Reply:
It's a lambda function in Golang